### PR TITLE
feature/content-container

### PR DESCRIPTION
**Pull request by seesleestak**
Adding ContentContainer. 

I was unable to add the `@hixme-ui/content-container` to hui-pages because I couldn't get it to publish. Hence the patch bump for all the components.

### CODE REVIEW

**Comment by billyxs** [comment]
We should publish from the master branch.  When in development, add the `@hixme-ui/content-containter` to the hui-pages package.json, and do lerna bootstrap.  Should be good to go then inside of hui-pages

File: packages/content-container/__tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { ThemeProvider } from '@hixme-ui/theme'
+import ContentContainer from '../src/index.js'
+
+describe('ContentContainer', () => {
+  it('Should have a h1 tag', () => {
```
Comment (billyxs) [inline packages/content-container/__tests__/index.test.js]: coming back to this?

**Comment by billyxs** [comment]
It looks like our hixmebot user doesn't have access to publish.  Only the hixmtech user. I'll look into adding permissions now that we've all switched to the hixmebot user.  If you want to use your personal npm, let me know

File: packages/content-container/__tests__/index.test.js (hunk 1 above)
Comment (seesleestak) [reply packages/content-container/__tests__/index.test.js]: Yeah, thanks

**Comment by billyxs** [comment]
I pushed the update to hui-pages.  lerna bootstrap worked fine

**Comment by seesleestak** [comment]
Ah, was just doing that 😆  I had tried that the first time but it wasn't working out for whatever reason.

**Comment by billyxs** [comment]
apologies @seesleestak. didn't want it to be confusing. I'll refrain from touching the beauty that is the content-container PR

**Comment by seesleestak** [comment]
Alright, I pushed up what might be a kinda janky usage of ContentContainer for the docs pages. Feel free to revert it if you don't dig it.

**Comment by billyxs** [comment]
janky, haha...no no. any progress is good progress

File: packages/content-container/__tests__/index.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { ThemeProvider } from '@hixme-ui/theme'
+import ContentContainer from '../src/index'
+
+const myChildren = 'Chosen one'
```
Comment (billyxs) [inline packages/content-container/__tests__/index.test.js]: you are...

**Pull request merged**